rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -185,4 +185,13 @@ abstract class CloudProviderBakeHandler {
     }
     return osVirtualizationSettings.baseImage
   }
+
+  /**
+   * packer variables names whose values will be replaced by '******'.
```

Comment:
Please capitalize the start of the comment and add that this only affects logging output.